### ISSUE TITLE
Rate limit headers are not coming for app registered in SharePoint

### ISSUE THREAD

**Issue by manjunathreddy1** [pnp/pnpcore#1159]
I have registered SharePoint app directly in SharePoint site (not as Azure AD app) using the pages appnegnew.aspx and appinv.aspx. I noticed that if I get the token for this app using following code rate limit headers are not coming up. But if register the app in Azure AD and use the following code rate limit headers are coming up. Please help to confirm if this the expected behavior or not.
var clientId = "";
var clientSecret = "";
var siteUrl = "";

PnP.Framework.AuthenticationManager authManager = new PnP.Framework.AuthenticationManager();
var sharePointAccessToken = authManager.GetACSAppOnlyContext(siteUrl, clientId, clientSecret).GetAccessToken()

Register the app directly in SharePoint site and not in Azure AD and use the above code.

**Comment by manjunathreddy1** [comment]
> @manjunathreddy1 : I need to cross check with engineering if there's a difference for Azure ACS. Do know that rate limit headers only are shown when you're at 80% or higher of your threshold. If in your testing with ACS you pushed until you got throttled and you've not seen rate limit headers before then they're definitely not included.

@jansenbe Yes. I'm aware of the point that rate limit headers will appear for the utilization above 80% but it doesn't appear when I connect to SharePoint site using app registered in SharePoint site but they are appearing when I use Azure AD app.

**Comment by jansenbe** [comment]
@manjunathreddy1 : we're cross checking this internally, I'll update here once I know more. Thanks for raising this.

**Comment by jansenbe** [comment]
@manjunathreddy1 : engineering confirmed we're sending rate limit headers, also when Azure ACS is used.

**Comment by jansenbe** [comment]
@manjunathreddy1 : are you good with closing this?

**Comment by manjunathreddy1** [comment]
> @manjunathreddy1 : are you good with closing this?

@jansenbe Yes it can be closed now. I created new SharePoint app and now the rate limit headers are appearing. Thanks.
